### PR TITLE
Enable voice chat feature for cross-platforms

### PR DESCRIPTION
**Pull request by raymondchams**
Enable voice chat feature for cross-platforms (Steam, Switch).

### CODE REVIEW

**Comment by nassosterz-ms** [comment]
Do we need to update anything here: https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/party-unreal-engine-oss-quickstart

Regarding the options for the chat features?

**Comment by nassosterz-ms** [comment]
Also the Readme has to be updated, I believe that with this change Steam and Switch go on GA.

File: SetUpPrivateOSS.ps1 [hunk 1]
```diff
@@ -10,27 +10,5 @@ Write-Host "Downloading Party and Multiplayer NuGet packages..."
...
-$partyBinary = Get-ChildItem $localPath -name -recurse Party.nro
-if (!$mlpBinary -or !$partyBinary) {
-	Write-Host "There was an error downloading the NuGet packages."
-	return
-}
-
-Write-Host "NuGet package downloads successful."
-
 Write-Host "Updating git submodules for private platforms..."
 git submodule update --recursive --init
```
Comment (PushpadantK) [inline SetUpPrivateOSS.ps1:14]: >update [](http://example.com/codeflow?start=14&length=6)

Did you update the platform specific files in Private repository?

**Comment by raymondchams** [comment]
> Also the Readme has to be updated, I believe that with this change Steam and Switch go on GA.

I don't think I have to update README for this changes. I think the updating document is enough.

File: SetUpPrivateOSS.ps1 (hunk 1 above)
Comment (raymondchams) [reply SetUpPrivateOSS.ps1:14]: No, this file changed by Bennie.
https://dev.azure.com/PlayFabInternal/Main/_git/MLP.UnrealOSS.Shared/commit/20dc2f508e10f54c5c96df7a63ef12a9c9128378?refName=refs%2Fheads%2Fdevelop&path=%2FSetUpPrivateOSS.ps1&_a=compare
Should I revoke this?

**Comment by nassosterz-ms** [comment]
The version on Readme has to change. Moreover:

"_The PlayFab Online Subsystem (PF OSS) v2 is currently generally available (GA) for GDK and in preview for Win64+Steam and Nintendo Switch. Preview programs give Microsoft and customers the opportunity to evaluate and understand the new feature before it becomes part of the standard feature set of PlayFab.

Preview SDKs and engine plugins might not support all the scenarios planned for the generally available (GA) release and are not necessarily fully compliant with the certification requirements for the target platform._"

This has to change to sth like:

"_The PlayFab Online Subsystem (PF OSS) v2 is currently generally available (GA) for GDK, Win64+Steam and Nintendo Switch"

**Comment by raymondchams** [comment]
> The version on Readme has to change. Moreover:
> 
> "_The PlayFab Online Subsystem (PF OSS) v2 is currently generally available (GA) for GDK and in preview for Win64+Steam and Nintendo Switch. Preview programs give Microsoft and customers the opportunity to evaluate and understand the new feature before it becomes part of the standard feature set of PlayFab.
> 
> Preview SDKs and engine plugins might not support all the scenarios planned for the generally available (GA) release and are not necessarily fully compliant with the certification requirements for the target platform._"
> 
> This has to change to sth like:
> 
> "_The PlayFab Online Subsystem (PF OSS) v2 is currently generally available (GA) for GDK, Win64+Steam and Nintendo Switch"

@PushpadantK Is this VOIP enable feature final changes? Can I remove preview now?

**Comment by raymondchams** [comment]
> Do we need to update anything here: https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/party-unreal-engine-oss-quickstart
> 
> Regarding the options for the chat features?

https://github.com/MicrosoftDocs/playfab-docs-pr/pull/1889

File: SetUpPrivateOSS.ps1 (hunk 1 above)
Comment (nassosterz-ms) [reply SetUpPrivateOSS.ps1:14]: Why was this file changed in the first place? The checks in the end are redundant, right?

**Comment by PushpadantK** [comment]
yes. but doc PR will be different than this one, but good callout.

---
In reply to: [1223040827](https://github.com/PlayFab/PlayFabMultiplayerUnreal/pull/9#issuecomment-1223040827) [](http://example.com/codeflow?ancestors=1223040827,1223040827)

**Review by nassosterz-ms — APPROVED** [review]

**Review by PushpadantK — APPROVED** [review]
:shipit:

**Pull request merged**